docs/docs/workflows/sbom.md

```diff
@@ -0,0 +1,88 @@
+# Consume Software Bill of Materials (SBOMs)
```

Comment:
```suggestion
# Consume software bill of materials (SBOMs)
```
I find both sentence-style and title-style capitalization equally often. As MS guide prefers sentence-style, I'd suggest this.

I'm also not sure whether the plural actually needs to be *bills*?
```suggestion
# Consume software bills of materials (SBOMs)
```

Reply:
I wen't with the first suggestion, double plural reads odd to me.